include/volk/volk_neon_intrinsics.h

```diff
@@ -286,7 +286,12 @@ static inline float32x4_t _neon_accumulate_square_sum_f32(float32x4_t sq_acc,
     aux = vmulq_f32(aux, val);
     aux = vsubq_f32(aux, acc);
     aux = vmulq_f32(aux, aux);
+#ifdef LV_HAVE_NEONV8
     return vfmaq_f32(sq_acc, aux, rec);
+#else
+    aux = vmulq_f32(aux, rec);
+    return vaddq_f32(aux, sq_acc);
+#endif
 }
 
-#endif /* INCLUDE_VOLK_VOLK_NEON_INTRINSICS_H_ */
```

Comment:
How did those newlines get lost? I suggest to keep them and make this PR even more concise.

Reply:
The newlines were lost in the previous commit. https://github.com/gnuradio/volk/commit/66a1b536bafe4d6e53ff88215231504ccc1893d2

My editor (I just use gedit on Ubuntu) will always insert a newline at the end of file. git hates missing a newline at end of file, so that's why I fixed the other two files.

Reply:
Ah. You re-added them. That's fine then.